docs/api/structures/trace-categories-and-options.md

```diff
@@ -0,0 +1,18 @@
+# TraceCategoriesAndOptions Object
+
+* `categoryFilter` String – is a filter to control what category groups
```

Comment:
nit: can just be 
`categoryFilter` String – a filter

Reply:
I just copied it from the old docs.